src/sagemaker/amazon/knn.py

```diff
@@ -108,6 +109,7 @@ def create_model(self, vpc_config_override=VPC_CONFIG_DEFAULT):
                 * 'Subnets' (list[str]): List of subnet ids.
                 * 'SecurityGroupIds' (list[str]): List of security group ids.
         """
+
```

Comment:
really? this newline is required? 😕

Reply:
not sure, but we had it in other places, so for consistency :)

Reply:
I think in most places we try not to include that newline: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html